src/Controllers/SettingsController.php

```diff
@@ -96,6 +96,49 @@ public function savePassword(Request $request): Response
         return $this->redirect->to('/settings/password');
     }
 
+    /**
+     * @return Response
+     */
+    public function theme(): Response
+    {
+        $themes = array_map(function ($theme) {
+            return $theme['name'];
+        }, config('themes'));
+
+        $currentTheme = $this->auth->user()->settings->theme;
+
+        return $this->response->withView(
+            'pages/settings/theme',
+            [
+                'settings_menu' => $this->settingsMenu(),
+                'themes' => $themes,
+                'current_theme' => $currentTheme
```

Comment:
I would have alligned the `=>` and added a `,` after the last one but thats only a minor stylistic change